nuxeo/values.yaml

```diff
@@ -45,7 +45,6 @@ nuxeo:
     username: nuxeo
     password: nuxeo
   elasticsearch:
-    clusterName: elasticsearch
```

Comment:
I'm not sure we can remove it.
It's used here: https://github.com/nuxeo/nuxeo-helm-chart/blob/master/nuxeo/templates/configmap.yaml#L33

By default, in Nuxeo we set the property to:
```
elasticsearch.clusterName=nuxeoCluster
```
I'm afraid this will set it to empty.

Reply:
we should remove this also in the configmap then, the cluster name is only used by TransportClient protocol (port 9300), when using the default RestClient the client (nuxeo) doesn't have to provide a cluster name.

Reply:
ok, let's use clusterName: nuxeoCluster it doesn't hurt.